src/algo/max-range/max-usable-range.cpp

```diff
@@ -0,0 +1,30 @@
+//// License: Apache 2.0. See LICENSE file in root directory.
+//// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#include "max-usable-range.h"
+
+
+using namespace librealsense::algo::max_range;
+
+float max_usable_range::get_max_range(float nest) const
+{
+    const float thermal = 74.5f;
```

Comment:
Please add a comment:
`// Based off of code in RS5-XXXX` (add ticket number)

Reply:
👍